authors/apps/comments/migrations/0001_initial.py

```diff
@@ -21,6 +21,9 @@ class Migration(migrations.Migration):
                 ('body', models.TextField(max_length=500)),
                 ('createdAt', models.DateTimeField(auto_now_add=True)),
                 ('updatedAt', models.DateTimeField(auto_now=True)),
+                ('highlight_start', models.IntegerField(null=True)),
```

Comment:
@d-rita dont merge migrations

Reply:
@ja-odur  I thought we are supposed to push one migration file

Reply:
@d-rita One migrations for your changes (per feature or story), but does not mean you update existing migration versions.

Reply:
makes sense 
@ja-odur  
@d-rita  had pushed her migration then I told her to delete it and make one migration for the whole thing my bad